packages/sources/layer2-sequencer-health/src/network.ts

```diff
@@ -49,7 +49,7 @@ export const checkSequencerHealth: NetworkHealthCheck = async (
   })
   const isHealthy = !!Requester.getResult(response.data, HEALTH_ENDPOINTS[network]?.responsePath)
   Logger.info(
-    `Health endpoint for network ${network} returned a ${
+    `[${network}] Health endpoint for network ${network} returned a ${
```

Comment:
is there a difference with `[${network}]` vs `${network}` or is this just for formatting reasons?

Reply:
Just formatting, figured the prefix `[...]` will make it easier to discern the different logs when grepping through.